packages/components/src/components/SquiggleEditor.tsx

```diff
@@ -15,6 +15,8 @@ export type SquiggleEditorProps = {
   defaultCode?: string;
   onCodeChange?(code: string): void;
   editorFontSize?: number;
+  seed?: string;
+  setSeed?(seed: string): void;
```

Comment:
These props are unused. Also, it's a problem that editors don't have re-rolls, e.g. in documentation we could have an example with randomness that would be hard to experiment with without random seeds.